README.md

```diff
@@ -7,41 +7,122 @@ A CNAB bundle for installing [riff](https://projectriff.io/)
+1. Use the duffle k8s driver to install riff. This will run riff bundle's invocationImage in k8s cluster using the service account created in the previous step. We will also set a couple of environment variables required by the k8s driver.
+
+### Detailed Instructions
+Follow the steps below for your choice of kubernetes cluster
+#### Minikube
+1. [Install minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+1. Start the minikube cluster
     ```
-    minikube config set embed-certs true
+    minikube start --memory=4096 --cpus=4 \
+    --kubernetes-version=v1.14.0 \
+    --vm-driver=hyperkit \
+    --bootstrapper=kubeadm \
+    --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
     ```
-1. Check that your kubeconfig file is configured to talk to that cluster:
+1. Confirm that your kubectl context is pointing to the new cluster
     ```
     kubectl config current-context
     ```
- 
-1. Add a duffle credential to allow this bundle to talk to your kubernetes cluster.
-    * Create a file named `myk8s.yaml` with the following content:
-     ```
-     name: myk8s
-     credentials:
-     - name: kubeconfig
-       source:
-         path: $HOME/.kube/config
-     ```
-    * Create a duffle credential using the above file:
-     ```
-     duffle credentials add myk8s.yaml
-     ```
-1. Download the latest snapshot of the bundle file:
+1. Install duffle by download the latest duffle release for your operating system from duffle's [release page](https://github.com/deislabs/duffle/releases), put it somewhere on your path, and make it executable, for example, `chmod +x /usr/local/bin/duffle`.
+1. Set environment variables required by duffle k8s driver and create the the service account
     ```
-    curl -O https://storage.googleapis.com/projectriff/riff-cnab/snapshots/riff-bundle-latest.json
+    export SERVICE_ACCOUNT=duffle-runtime
+    export KUBE_NAMESPACE=duffle
+    kubectl create namespace $KUBE_NAMESPACE
+    kubectl create serviceaccount "${SERVICE_ACCOUNT}" -n "${KUBE_NAMESPACE}"
+    kubectl create clusterrolebinding "${SERVICE_ACCOUNT}-cluster-admin" --clusterrole cluster-admin --serviceaccount "${KUBE_NAMESPACE}:${SERVICE_ACCOUNT}"
     ```
-1. Install riff
+1. install riff
     ```
-    duffle install myriff riff-bundle-latest.json --bundle-is-file --credentials myk8s --insecure
+    duffle install myriff https://storage.googleapis.com/projectriff/riff-cnab/snapshots/riff-bundle-latest.json --bundle-is-file -s node_port=true -d k8s
     ```
+    where `node_port=true` parameter changes all service types to NodePort from LoadBalancer
+    and `-d k8s` uses the duffle kubernetes driver to run the installer image in kubernetes cluster
 1. You should now be able to see riff components installed on your kubernetes cluster:
     ```
     kubectl get pods --all-namespaces
     ```
+
+#### Docker for Desktop
```

Comment:
let's not go overboard repeating documentation. It creates a maintenance burden and is likely to go out of date. If someone is reading this doc trying to install riff rather than reading projectriff.io let's assume they understand the basics of creating a k8s cluster and are looking for details about what makes the riff cnab bundle unique.